server/app/sharing_test.go

```diff
@@ -0,0 +1,114 @@
+package app
+
+import (
+	"database/sql"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/uuid"
+	"github.com/mattermost/focalboard/server/auth"
+	"github.com/mattermost/focalboard/server/model"
+	"github.com/mattermost/focalboard/server/services/config"
+	st "github.com/mattermost/focalboard/server/services/store"
+	"github.com/mattermost/focalboard/server/services/store/mockstore"
+	"github.com/mattermost/focalboard/server/services/webhook"
+	"github.com/mattermost/focalboard/server/ws"
+	"github.com/mattermost/mattermost-server/v5/services/filesstore/mocks"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetSharing(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	cfg := config.Configuration{}
+	store := mockstore.NewMockStore(ctrl)
+	auth := auth.New(&cfg, store)
+	sessionToken := "TESTTOKEN"
+	wsserver := ws.NewServer(auth, sessionToken)
+	webhook := webhook.NewClient(&cfg)
+	app := New(&cfg, store, auth, wsserver, &mocks.FileBackend{}, webhook)
```

Comment:
Once this PR get merged (hopefully tomorrow) -https://github.com/mattermost/focalboard/pull/422
This can be update to...
```suggestion
	th := SetupTestHelper(t)
```

Reply:
Sure! No problem :smile: